modules/overrides/userconfigurable/api/http.go

```diff
@@ -82,13 +85,18 @@ func (a *UserConfigOverridesAPI) PostHandler(w http.ResponseWriter, r *http.Requ
 		return
 	}
 
+	skipConflictingOverridesCheck := false
```

Comment:
Do we want to pull this from config instead of a default value here?

Reply:
We read the value from config in `api.go`, the http layer is just parsing the header and passing the value to `UserConfigOverridesAPI.set`.